tools/travis/before_script.sh

```diff
@@ -70,11 +70,13 @@ else
                       "--with-mysql-build=$PWD/vendor/mysql/release")
       ;;
     mysql-8.0)
-      boost_archive=boost_1_69_0.tar.gz
-      curl -L -O https://dl.bintray.com/boostorg/release/1.69.0/source/${boost_archive}
+      #boost_archive=boost_1_69_0.tar.gz
```

Comment:
Is this comment need?

Reply:
At first, the download issue will be fixed in upstream, then I think to get them back.(so I've treated them as comments)
Now, it seems better to remove old stuffs. so I'll remove them.